ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java

```diff
@@ -63,7 +63,7 @@
     List<Integer> active_validator_indices = new ArrayList<>();
     IntStream.range(0, validators.size())
         .parallel()
-        .forEachOrdered(
+        .forEach(
```

Comment:
idk what the difference between forEach and forEachOrdered is but to me this seems like using parallel processing to add to the same list could create concurrency issues.

Reply:
So forEachOrdered guarantees the process order is the same as the encountered order. I'd imagine it has worse performance as compared to forEach when doing parallel stream processing. In this case, because all that we're doing is adding indices that meet a certain condition to a list, output order isn't important. 

tl;dr I think we're safe here.

Reply:
agreed on both counts.  good tweak